microsoft/PowerToys
# Remove the indexer plugin from constant search plugins

## Summary of the Pull Request

_What is this about?_
Remove the indexer plugin from constant search plugins. 

## PR Checklist
* [X] Applies to #8165

## Info on Pull Request

_What does this include?_
- Remove logic that simplifies indexer search query
- Return empty list in Query

## Validation Steps Performed

_How does someone test & validate?_
- Check if the indexer plugin works. It can be done by searching for something simple as on the image below. Indexer's results have `Search: {location of the file}` description.

![image](https://user-images.githubusercontent.com/17161067/101384292-e911c980-38c2-11eb-9ee0-543bbd21ce1a.png)

- Use artificial `Thread.Sleep` to make sure that the indexer results are fetched with delay plugin
